Web/starter_project_iconics/styles/globals.css

```diff
@@ -1,7 +1,14 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;600;700;800&family=Poppins:ital,wght@0,200;0,300;0,400;0,700;0,800;1,600&display=swap');
```

Comment:
i am not sure if this is the right place to import this
i think there might be a javascript way of doing it without having to import the whole font in the glbal css
please check if that is the case but otherwise it could be fine